### DISCUSSION TITLE
Future of Home Assistant Formula One API

### DISCUSSION THREAD

**Discussion by delzear** [delzear/hass-formulaoneapi#17] [Q&A]
I have been talking with Ergast API to still their service but we will need a caching server. I have the infrastructure to build and maintain this service but I will need help modifying the Python.

I have a Web Api that we can test with I will need help for the sensor. Is someone available to help with this part?

**Comment by delzear** [comment]
@huizebruin @LRvdLinden Would one of you be able to help me?

**Reply by huizebruin** [reply]
hello @delzear .

i would love it to do but,
I'm a beginner with code for home assistand and i now how to change some frontend code but python i don't now how to do that.

**Reply by LRvdLinden** [reply]
@delzear what for help?

**Reply by delzear** [reply]
Ergast asked me to cache the requests to minimize the amount of traffic on his server. I will route all traffic from this integration to my server and I want to make a smart schedule for the requests. 

I will make the server side of it but the python is harder for me.

**Reply by OdinMegan** [reply]
where to find the frontend file ? to edit

**Reply by huizebruin** [reply]
https://github.com/delzear/hass-formulaoneapi/blob/master/frontend.md

**Reply by OdinMegan** [reply]
that one i found, but where to place inside the HA setup ?

**Reply by delzear** [reply]
You can place the content in a markdown card in a dashboard

**Reply by OdinMegan** [reply]
Great it works, many thnx sorry i am new to HA :)

**Reply by OdinMegan** [reply]
is there also the posibility to create the weekend time table ?

**Reply by massimoha** [reply]
Hey guys, I am also new and have placed the data into markdown cards a week ago but my cards display poorly with large gaps in them...just asking for assistance in what i am possibly doing wrong for them to display this way? see attahced photos...

![F1 Cards - 1](https://user-images.githubusercontent.com/111345084/191877303-944f3916-0668-4dd3-b6ff-133be553b47c.PNG)
![F1 cards - 2](https://user-images.githubusercontent.com/111345084/191877316-b6c09afa-e484-4841-8063-3c6346b9851e.PNG)
